rules/server/server.go

```diff
@@ -0,0 +1,129 @@
+// Package rulesserver provides primitives to interact with the openapi HTTP API.
+//
+// Code generated by github.com/deepmap/oapi-codegen version v1.8.3 DO NOT EDIT.
+package rulesserver
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/deepmap/oapi-codegen/pkg/runtime"
+	"github.com/go-chi/chi/v5"
+)
+
+// ServerInterface represents all server handlers.
+type ServerInterface interface {
```

Comment:
I wished someday we have gRPC defined APIs that by default expose  HTTP route too (like https://github.com/twitchtv/twirp) 😢